shell/platform/linux/public/flutter_linux/fl_platform_views.h

```diff
@@ -0,0 +1,224 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
```

Comment:
Can you split this file into `fl_platform_view.h` and `fl_platform_view_factory.h`? It's easier to understand with one class per header.